conmon-rs/common/proto/conmon.capnp

```diff
@@ -120,4 +121,16 @@ interface Conmon {
     }
 
     setWindowSizeContainer @5 (request: SetWindowSizeRequest) -> (response: SetWindowSizeResponse);
+
+    ###############################################
+    # CreateNamespaces
+    struct CreateNamespacesRequest {
+        metadata @0 :Data;
+    }
+
+    struct CreateNamespacesResponse {
+        path @0 :Text;
```

Comment:
Is this enough or should we abstract even more? Right now we can expect mounted namespaces following the naming convention under this path. We could also have dedicated types mapped around it.